packages/itwinui-variables/.gitignore

```diff
@@ -0,0 +1,2 @@
+index.css
+index.css.*
```

Comment:
What it ignores?

Reply:
Do we even need to generate map?

Reply:
Removed sourcemap from prod build. I think it can be useful for development